src/components/Navigation.tsx

```diff
@@ -20,18 +22,21 @@ export const Navigation: React.FC = () => {
         <li className="hidden sm:block">ロゴ</li>
         {LABELS.map(({ href, label, icon }) => {
           const isActive = pathname === href;
+          const activeColorClass = cc([
```

Comment:
activeColorClassだとアクティブな状態のクラスのみな印象になるので、ラベルカラークラスとか？の方が良い気がします。
```suggestion
          const activeColorClass = cc([
```

Reply:
@KohnoTaiyo レビューありがとうございます！
クラス名をlabelColorClassにしてプッシュしました。